includes/apple-exporter/builders/class-components.php

```diff
@@ -486,13 +486,6 @@ private function convert_ids_to_identifiers( &$component ) {
 		$component['text'] = preg_replace_callback(
 			'/\bid=["\'](.*?)["\']/',
 			function ( $matches ) use ( &$component, &$identifiers ) {
-				// If 'id' starts with a digit, it's skipped,
```

Comment:
Have you confirmed that Apple will not reject items with digits at the start of IDs?

Reply:
@kevinfodness it works in News Preview and the Simulator, but I have not tested it beyond that. Looks like my local is pushing to your channel, so I'll test it there.

Reply:
works on your channel.